src/components/Message/MessageSimple/ReactionList.tsx

```diff
@@ -63,13 +70,41 @@ const Icon: React.FC<
   const ReactionIcon =
     supportedReactions.find((reaction) => reaction.type === type)?.Icon ||
     Unknown;
+
+  const scale = useSharedValue(0, false);
```

Comment:
heads up, in 2.0.0 there's no 2nd param you can pass in for this hook

Reply:
Yeah need it here for now as it's not the default and kept getting rebuilt.